### PR TITLE
Update Documentation for TripletMarginLoss

### PR DESCRIPTION
**Pull request by danielzhangau**
This PR updates the documentation for `TripletMarginLoss` in `torch.nn`. The previous version of the documentation didn't mention the parameter `eps` used for numerical stability. 

This PR does the following:
1. Describes the purpose and use of the `eps` parameter in the `TripletMarginLoss` class documentation.
2. Includes `eps` in the example usage of `TripletMarginLoss`.

Please review this update for the completeness with respect to the `TripletMarginLoss` functionality. If there are any issues or further changes needed, please let me know.

### CODE REVIEW

**Review by mikaylagawarecki — COMMENTED** [review]
Thanks! The docs fixes look ok but could you please fix the failing lint



To fix the failing lint job running `setup_lint` and `quicklint` from your local pytorch directory and following the corresponding instructions should be sufficient, see [here](https://github.com/pytorch/pytorch/blob/4721553431e968ee5e90753346d92252cce98e84/Makefile#L25-L34)

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -1471,7 +1475,7 @@ class TripletMarginLoss(_Loss):
 
     Examples::
 
-    >>> triplet_loss = nn.TripletMarginLoss(margin=1.0, p=2)
+    >>> triplet_loss = nn.TripletMarginLoss(margin=1.0, p=2, eps=1e-6)
```
Comment (mikaylagawarecki) [inline torch/nn/modules/loss.py:1478]: nit: maybe use a different value from the default :)

Comment (danielzhangau) [reply torch/nn/modules/loss.py:1478]: Thanks @mikaylagawarecki for your feedback. I was considering changing the `eps` value to `1e-7` in the example. Would that be appropriate, or do you suggest another value?

**Review by mikaylagawarecki — APPROVED** [review]
Thanks!
